test/expected/misc.out

```diff
@@ -197,3 +197,6 @@ EXPLAIN SELECT city FROM test_customers ORDER BY 1;
                ->  Seq Scan on test_customers
 (6 rows)
 
+-- Reject unsupported column types during AID labeling
+SECURITY LABEL FOR pg_diffix ON COLUMN test_customers.discount IS 'aid';
```

Comment:
wouldn't this test be more tight, if the attempt was to set a column with a supported type? Also, the comment above is not in sync with the assertion.

Reply:
Specifying a supported type is the happy path, which is tested everywhere else. I wanted to test here specifically that unsupported types are rejected.
I don't understand your last comment. It looks to me that they are in sync.

Reply:
"unsupported column types" vs "requires direct access level"? the case in `admin.out` looks good but this seems off.

Reply:
or maybe this is just a leftover, and you only want the `admin.out` one?

Reply:
Ah, ok, I get it now :). You're right, I committed some wrong stuff by mistake. Thanks for catching it!

Reply:
heh, it looked like a quite legit test, hence my confusion :)